### DISCUSSION TITLE
An unhandled exception occurred while processing the request.

### DISCUSSION THREAD

**Discussion by olulekeokewale** [community/community#19162] [Projects and Issues] [answered]
I just tried to run sample Abp.BookStore Solution and I got this error message
AbpException: Could not find the bundle file '/libs/abp/core/abp.css' for the bundle 'Basic.Global'!

![image](https://user-images.githubusercontent.com/49058593/175009882-f7b74f16-5cb6-4262-a3c0-11064d01f96d.png)
I have npm installed and yet its telling me npm not installed when running abp install-libs

**Comment by melvincwng** [comment, marked as answer]
Hi @olulekeokewale, to fix this issue, try running the following commands in Powershell: 

1. Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process
2. npm install -g npm-windows-upgrade
3. npm-windows-upgrade

This issue probably occurred because you need to update your npm version in Windows.

Hope this solution works for you. Please check out the references attached below for more info.

References:
[Issue 12995 in ABP framework](https://github.com/abpframework/abp/issues/12995)

[Stackoverflow](https://stackoverflow.com/questions/18412129/how-can-i-update-npm-on-windows)

**Reply by melvincwng** [reply]
@olulekeokewale, if you find this reply helpful, please kindly 'Mark as answer' so that others are aware of the solution, should they face similar problems in the future.

Thank you for your cooperation and have a great day 😊

**Reply by olulekeokewale** [reply]
Very helpful. Thank you

**Reply by melvincwng** [reply]
@olulekeokewale, no problem! Glad that I was able to help you 😌
